swarm/storage/mru/signedupdate.go

```diff
@@ -102,7 +105,7 @@ func (r *SignedResourceUpdate) toChunk() (*storage.Chunk, error) {
 	// For efficiency, data is serialized during signature and cached in
 	// the binaryData field when computing the signature digest in .getDigest()
 	if r.signature == nil || r.binaryData == nil {
-		return nil, NewError(ErrInvalidSignature, "newUpdateChunk called without a valid signature or payload data. Call .Sign() first.")
+		return nil, NewError(ErrInvalidSignature, "toChunk called without a valid signature or payload data. Call .Sign() first.")
```

Comment:
Why `toChunk`?

Reply:
Gives you a chunk out of a `SignedResourceUpdate` This function was renamed to this as per the request (Viktor's?) during the other PR's review